src/test/java/core/basesyntax/sarvice/impl/FruitTransactionParserServiceImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.sarvice.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.FruitTransactionParserServiceImpl;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionParserServiceImplTest {
+    private static FruitTransactionParserServiceImpl parser;
+    private static List<FruitTransaction> expected;
+
+    @BeforeClass
+    public static void beforeClass() {
+        parser = new FruitTransactionParserServiceImpl();
+        expected = new LinkedList<>();
+    }
+
+    @Before
+    public void setUp() {
+        expected.add(new FruitTransaction(
+                FruitTransaction.Operation.BALANCE, "banana", 20));
+        expected.add(new FruitTransaction(
+                FruitTransaction.Operation.BALANCE, "apple", 100));
+        expected.add(new FruitTransaction(
+                FruitTransaction.Operation.SUPPLY, "banana", 100));
```

Comment:
You did not correct Nastya's comment here

Reply:
I don't understand then. Can you explain to me in more detail, please.

Reply:
If my corrections are wrong, can you explain more, please?